elide-standalone/src/test/java/com/yahoo/elide/standalone/ElideStandaloneTest.java

```diff
@@ -89,79 +102,122 @@ public void shutdown() throws Exception {
         elide.stop();
     }
 
+    @Test
+    public void testJsonAPIPlayer() {
+        given()
+        .contentType(JSONAPI_CONTENT_TYPE)
+        .accept(JSONAPI_CONTENT_TYPE)
+        .body(
+                datum(
+                        resource(
+                                type("player"),
+                                id("ready-player-1"),
+                                attributes(
+                                        attr("name", "player1"),
+                                        attr("countryCode", "USA"),
+                                        attr("playerCountry", "USA"),
+                                        attr("highScore", 100)
+                                        //                        attr("createdOn", "2020-01-01"); still need to be tested with date
```

Comment:
Please check the spring PR. I think we were able to successfully add it there. I think it should be like this 2020-01-01T00:00Z